### PR TITLE
feat(mutator): Object literal mutator

### PR DESCRIPTION
**Pull request by tommilligan**
As suggested by @kmdrGroch in #1078, a mutator which removes all properties from an object literal during initialization.

Working TS + JS implementation, please let me know any additional cases/requirements you'd like me to test.

### CODE REVIEW

**Comment by kmdrGroch** [comment]
@tommilligan is it removing all props at the same time or removes one by one?

**Comment by kmdrGroch** [comment]
And covers objects like shown in https://github.com/stryker-mutator/stryker/issues/1057

**Comment by tommilligan** [comment]
All props simultaneously. I would expect other mutators to mutate the values if necessary (e.g. if a property is a string, for `StringMutator` to pick it up).

File: packages/stryker-mutator-specification/src/ObjectMutatorSpec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import ExpectMutation from './ExpectMutation';
+
+export default function ObjectMutatorSpec(
+  name: string,
+  expectMutation: ExpectMutation
+) {
+  describe('ObjectMutator', () => {
+    it('should have name "Object"', () => {
```
Comment (nicojs) [inline packages/stryker-mutator-specification/src/ObjectMutatorSpec.ts]: `"Object"` mutator sounds pretty generic. I think `ObjectLiteral` would be a better fit.

Comment (nicojs) [reply packages/stryker-mutator-specification/src/ObjectMutatorSpec.ts]: Could we also add a test for:

```js
{ ['key']: 'value' } // mutated to { }
```

What do you think?

Comment (tommilligan) [reply packages/stryker-mutator-specification/src/ObjectMutatorSpec.ts]: @nicojs good point on the name - have amended both points

**Comment by nicojs** [comment]
Thank @tommilligan 👍

**Pull request merged**